dizoo/mujoco/config/walker2d_trex_sac_config.py

```diff
@@ -78,5 +84,10 @@
     ),
     replay_buffer=dict(type='naive', ),
 )
-walker2d_trex_sac_default_create_config = EasyDict(walker2d_trex_sac_default_create_config)
-create_config = walker2d_trex_sac_default_create_config
+walker2d_trex_sac_create_config = EasyDict(walker2d_trex_sac_create_config)
+create_config = walker2d_trex_sac_create_config
+
+
+if __name__ == "__main__":
```

Comment:
add line # or you can enter `ding -m serial_trex -c walker2d_trex_sac_config.py -s 0`